src/main/java/com/google/sps/servlets/RetrieveActivitiesServlet.java

```diff
@@ -0,0 +1,82 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+
+import com.google.sps.data.Activity;
+import com.google.sps.data.Activity.Category;
+import static com.google.sps.data.ActivitiesUtility.getActivities;
+
+import java.util.List;
+import java.util.ArrayList;
+
+/** 
+* Servlet that retrieves actvities of a specified category.
+*/
+@WebServlet("/retrieve-activities")
+public class RetrieveActivitiesServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Category activityCategory = getActivityCategory(request);
+    List<Activity> activities;
+
+    switch(activityCategory) {
```

Comment:
This switch statement can be removed. getActivities(activityCategory) should be sufficient.

Reply:
Fixed it.